app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -86,7 +86,6 @@ def destroy_relation(resource, relation, value)
   private
 
   def remove_linked_set_member_subjects(resource, set_member_subjects)
-    SubjectWorkflowCount.where(set_member_subject: set_member_subjects).delete_all
```

Comment:
Could but then we can't set subject_id  to not null
On Sep 8, 2015 11:34 AM, "Campbell Allen" <notifications@github.com> wrote:

> In app/controllers/api/v1/subject_sets_controller.rb
> <https://github.com/zooniverse/Panoptes/pull/1337#discussion_r38911287>:
>
> > @@ -86,7 +86,6 @@ def destroy_relation(resource, relation, value)
> >    private
> >
> >    def remove_linked_set_member_subjects(resource, set_member_subjects)
> > -    SubjectWorkflowCount.where(set_member_subject: set_member_subjects).delete_all
>
> are we keeping these around for reference?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zooniverse/Panoptes/pull/1337/files#r38911287>.
>